src/test/scala/com.lightbend.paradox/unidoc/UnidocPluginSpec.scala

```diff
@@ -4,5 +4,135 @@
 
 package com.lightbend.paradox.unidoc
 
-class UnidocPluginSpec {
+import com.lightbend.paradox.markdown.Writer
+
+class UnidocPluginSpec extends MarkdownBaseSpec {
+  val rootPackage = "akka"
+
+  val allClasses = Array(
+    "akka.actor.ActorRef",
+    "akka.actor.typed.ActorRef",
+    "akka.cluster.client.ClusterClient",
+    "akka.cluster.client.ClusterClient$",
+    "akka.dispatch.Envelope",
+    "akka.http.javadsl.model.sse.ServerSentEvent",
+    "akka.http.javadsl.marshalling.Marshaller",
+    "akka.http.javadsl.marshalling.Marshaller$",
+    "akka.http.scaladsl.marshalling.Marshaller",
+    "akka.http.scaladsl.marshalling.Marshaller$",
+    "akka.stream.javadsl.Source",
+    "akka.stream.javadsl.Source$",
+    "akka.stream.scaladsl.Source",
+    "akka.stream.scaladsl.Source$",
+    "akka.stream.javadsl.Flow",
+    "akka.stream.javadsl.Flow$",
+    "akka.stream.scaladsl.Flow",
+    "akka.stream.scaladsl.Flow$"
+  )
+
+  override val markdownWriter = new Writer(
+    linkRenderer = Writer.defaultLinks,
+    verbatimSerializers = Writer.defaultVerbatims,
+    serializerPlugins = Writer.defaultPlugins(
+      Writer.defaultDirectives ++ Seq(
+        (_: Writer.Context) => new UnidocDirective(allClasses)
+      )
+    )
+  )
+
+  implicit val context = writerContextWithProperties(
+    "scaladoc.akka.base_url" -> "https://doc.akka.io/api/akka/2.5",
+    "scaladoc.akka.http.base_url" -> "https://doc.akka.io/api/akka-http/current",
+    "javadoc.akka.base_url" -> "https://doc.akka.io/japi/akka/2.5",
+    "javadoc.akka.http.base_url" -> "https://doc.akka.io/japi/akka-http/current",
+  )
+
```

Comment:
Links from the expected values of the test cases are here, which all confirmed valid:

https://doc.akka.io/api/akka/2.5/akka/dispatch/Envelope.html
https://doc.akka.io/japi/akka/2.5/?akka/dispatch/Envelope.html

https://doc.akka.io/japi/akka/2.5/?akka/stream/javadsl/Flow.html
https://doc.akka.io/api/akka/2.5/akka/stream/scaladsl/Flow.html

https://doc.akka.io/api/akka/2.5/akka/actor/ActorRef.html
https://doc.akka.io/japi/akka/2.5/?akka/actor/ActorRef.html

https://doc.akka.io/api/akka/2.5/akka/cluster/client/ClusterClient$.html
https://doc.akka.io/japi/akka/2.5/?akka/cluster/client/ClusterClient$.html

https://doc.akka.io/japi/akka/2.5/?akka/stream/javadsl/Source.html
https://doc.akka.io/api/akka/2.5/akka/stream/scaladsl/Source.html

https://doc.akka.io/japi/akka/2.5/?akka/stream/javadsl/Flow.html
https://doc.akka.io/api/akka/2.5/akka/stream/scaladsl/Flow.html

https://doc.akka.io/japi/akka-http/current/?akka/http/javadsl/marshalling/Marshaller.html
https://doc.akka.io/api/akka-http/current/akka/http/scaladsl/marshalling/Marshaller.html